pypesto/optimize/optimizer.py

```diff
@@ -68,9 +81,11 @@ def time_decorator(minimize):
     the wall-clock time.
     """
 
-    def wrapped_minimize(self, problem, x0, id, history_options=None):
+    def wrapped_minimize(self, problem, x0, id, allow_failed_starts,
```

Comment:
These wrappers are getting a little complicated, as we have not at least two arguments that are not needed for the "minimization" task, just in the wrappers. I suggest reformulating this as functions of Optimizer at some point, but fine for the moment.